code/implementation/docker/common/src/main/java/io/cattle/platform/docker/transform/DockerTransformerImpl.java

```diff
@@ -172,7 +172,7 @@ void setNetworkMode(Instance instance, ContainerConfig containerConfig, HostConf
                     NETWORK_MODE_HOST.equals(inspectNetMode) ||
                     NETWORK_MODE_NONE.equals(inspectNetMode)) {
                 netMode = inspectNetMode;
-            } else if (StringUtils.isBlank(inspectNetMode)) {
+            } else if (NETWORK_MODE_DEFAULT.equals(inspectNetMode) || StringUtils.isBlank(inspectNetMode)) {
```

Comment:
Would it be backwards compatible with docker 1.7 if some users still use it? or do we force the upgrade?